db/schema.rb

```diff
@@ -2156,125 +2159,225 @@
   add_foreign_key "users", "staff_roles", on_delete: :restrict
   add_foreign_key "weather_observations", "weather_stations", on_delete: :cascade
 
-  create_view "annual_change_in_electricity_out_of_hours_uses", sql_definition: <<-SQL
```

Comment:
Schema messed up again. Is it just me?

Reply:
No it happens for me too, still not sure why.